3C

0shiros opened the pull request:
- Camera follow player with cinemachine
- Control player made with unity input system (zqsd)
- player can shoot (a bullet children) or hit(debug.log)

0shiros committed:
3C

- Camera follow player with cinemachine
- Control player made with unity input system (zqsd)
- player can shoot (a bullet children) or hit(debug.log)

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Attack/HitMele.cs:
retire le using NUnit.Framework.

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Attack/Shoot.cs:
NUnit pareil

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Bullet/Bullet.cs:
pareil

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Attack/HitMele.cs:
renomme bien toutes tes actions par le bon nom.

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Attack/Shoot.cs:
pareil pour l'input, il faut que les variables ai un nom explicit

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Attack/Shoot.cs:
Petit tips : quand tu fais transform.quelquechose c'est comme si tu faisais GetComponent<Transform> donc ça prends des performances. à l'awake store le dans une variable comme suit : _transform = transform. Et utiilises la à la place des transforms

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Bullet/Bullet.cs:
pareil pour le transform ici

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Movements/PlayerMovements.cs:
pareil pour le NUnit

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Movements/PlayerMovements.cs:
pareil pour le nom de la variable de l'input

DoriannX commented on Instance2-BattleArena/Assets/Scripts/Mechanics/Movements/PlayerMovements.cs:
Pareil pour le transform

0shiros committed:
Changed using NUnit, name variable, transform get

- Replace Nunit.framework by UnityEngine.Assert
- Set variable to stock transform
- Rename variables of input and some others
